src/en/charms-storage-ceph.md

```diff
@@ -0,0 +1,123 @@
+Title: Installing Ceph
```

Comment:
There is certainly space for a standalone tutorial covering this and more. The purpose of this page was merely to set up a minimal Ceph cluster so we could use it as a guided example. The simpler it is and the less detail it goes into the better, as a) people will need to follow it and b) we will need to maintain it in docs. It could easily link to a tutorial here if one were to exist